src/Chapter1.hs

```diff
@@ -631,8 +641,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = x1 + x2
+  where
+    x1 = abs n `rem` 100 `div` 10
```

Comment:
When using functions written in the infix form, it is important to be cautious with the precedence as it is not explicitly determined for each function (unlike operators). So, it would be better to put braces in this case, to be sure that it will work as you expected.

Also, usually `rem` is used in pair with `quot`, and `mod` with `div`. You can also find such interesting functions like `divMod` or `quotRem` that could also be useful in here 🙂